aries_cloudagent/protocols/present_proof/v2_0/formats/dif/handler.py

```diff
@@ -292,6 +293,11 @@ async def create_pres(
                 # For now, setting to 1000
                 max_results = 1000
                 records = await search.fetch(max_results)
+                # sort records by issuanceDate in reverse_order
+                records.sort(
+                    key=lambda v: dateutil_parser(v.cred_value.get("issuanceDate")),
+                    reverse=True,
+                )
```

Comment:
I think this needs to be a little more tolerant of errors. It looks like `dateutil_parser` will throw an error if the issuanceDate is undefined, and it probably would if the date's not in a supported format.

Reply:
The `issuanceDate` is required and must be in `RFC3339` format. This is enforced in `./issue_credential/v2_0/formats/ld_proof/handler.py` when receiving and storing credential with `./vc/vc_ld/models/credential.py -> VerifiableCredential`. I can validate against `RFC3339DateTime` before sorting but I think it will be a redundant check.
https://www.w3.org/TR/vc-data-model/#issuance-date

Reply:
Okay, I was just thinking to catch exceptions if any occur.

Reply:
I get your point regarding catching exceptions, I will add that in.

Reply:
Exception handling added.